src/controllers/admin/index.js

```diff
@@ -0,0 +1,7 @@
+// Welcome page controller
+
```

Comment:
whats the use of this controller

Reply:
this is like the index route of the super admin, should I remove it?